build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -26,6 +26,7 @@
 LARGE = [
     tf_models.MINILM_L12_H384_UNCASED_INT32_SEQLEN128,
     tf_models.BERT_FOR_MASKED_LM_FP32_SEQLEN512,
+    tf_models.EFFICIENTNET_V2_S_FP32,
```

Comment:
Is EfficientNet a large model? (It looks like we run the int8 versions of them on mobile). Maybe we should put them into the small model group?

Reply:
EfficientNet comes in 3 sizes and will fall into either SMALL or LARGE depending on the size. The small version is about the size of MobileBert and the Large version is about the size of BERT-base. Ideally it should be tested on both mobile and server.

Reply:
We probably need another way to group these models. Currently I just tried to replicate our legacy benchmark suites.

If this is a small model (suitable to run on mobiles), we can put in in the SMALL group and add this model for CUDA (https://github.com/iree-org/iree/blob/main/build_tools/python/benchmark_suites/iree/cuda_benchmarks.py#L39); otherwise I'm fine with it.

Reply:
Oh, I didn't notice the auto-merge is enabled...